templates/fenix_events_amplitude_v1.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
+  * EXCEPT (submission_date)
+FROM
+  `moz-fx-data-derived-datasets.telemetry.fenix_events_v1`
```

Comment:
this should not reference the fully qualified table unless it's a view:
```suggestion
  fenix_events_v1
```

Reply:
👍